source/opt/reflect.h

```diff
@@ -61,7 +61,9 @@ inline bool IsSpecConstantInst(SpvOp opcode) {
 }
 inline bool IsTerminatorInst(SpvOp opcode) {
   return (opcode >= SpvOpBranch && opcode <= SpvOpUnreachable) ||
-         (opcode == SpvOpTerminateInvocation);
+         (opcode == SpvOpTerminateInvocation ||
+          opcode == SpvOpTerminateRayKHR ||
+          opcode == SpvOpIgnoreIntersectionKHR);
```

Comment:
Could this call `spvOpcodeIsBlockTerminator` instead of checking the opcode itself?

Reply:
yes, they are checking the same 10 opcodes. Done.